bundles/org.openhab.binding.pihole/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="pihole"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="server">
+		<label>Pi-hole Server</label>
+		<description>Sample thing for PiHole Binding</description>
+
+		<channels>
+			<channel id="domains-being-blocked" typeId="number-channel">
+				<label>Domains Blocked</label>
+				<description>This thing represents a Pi-hole server and is used for the PiHole Binding.</description>
```

Comment:
This seems to be the description that belongs to the pi-hole server, not to the domains blocked.

Reply:
yeah, stupid mistake. I've fixed it.